frontend/src/App.tsx

```diff
@@ -1,14 +1,19 @@
 import React from 'react';
 import useAccessToken from "./hooks/useAccessToken";
 import Homepage from "./components/Homepage";
+import {BrowserRouter, Route, Routes} from "react-router-dom";
 
 function App() {
 
-  const {token} = useAccessToken()
+    const {token} = useAccessToken()
```

Comment:
I think it'd be great to specify the type of token.
A `mapBoxToken` in this case.
I thought that you're using a JWT token for a moment.